.github/workflows/.test-prepare.yml

```diff
@@ -16,20 +16,6 @@ jobs:
     outputs:
       matrix: ${{ steps.set.outputs.matrix }}
     steps:
-      -
-        name: Checkout
-        uses: actions/checkout@v3
-      -
-        name: Create matrix
+      - name: Create matrix
         id: set
-        uses: actions/github-script@v6
-        with:
-          script: |
-            let matrix = ['graphdriver'];
-            if ("${{ contains(github.event.pull_request.labels.*.name, 'containerd-integration') || github.event_name != 'pull_request' }}" == "true") {
-              matrix.push('snapshotter');
-            }
-            await core.group(`Set matrix`, async () => {
-              core.info(`matrix: ${JSON.stringify(matrix)}`);
-              core.setOutput('matrix', JSON.stringify(matrix));
-            });
+        run: echo 'matrix=["graphdriver", "snapshotter"]' >> "$GITHUB_OUTPUT"
```

Comment:
Since it's unconditional there is no need to run a script, you can:

```yaml
outputs:
  matrix:
    - graphdriver
    - snapshotter
```

Reply:
Thanks, fixed! I tried it initially but didn't get the job/step syntax right and then forgot about getting back to it.

Reply:
Oh, I think I actually can't:
```
error parsing called workflow ".github/workflows/windows-2022.yml" -> "./.github/workflows/.test-prepare.yml" : every step must define a `uses` or `run` key
```

Reply:
ugh